keyboards/clueboard/60/config.h

```diff
@@ -53,9 +53,9 @@
 #define PREVENT_STUCK_MODIFIERS
 
 /* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
-//#define LOCKING_SUPPORT_ENABLE
+#define LOCKING_SUPPORT_ENABLE
 /* Locking resynchronize hack */
-//#define LOCKING_RESYNC_ENABLE
+#define LOCKING_RESYNC_ENABLE
```

Comment:
Can you place these in your local `config.h`, unless @skullydazed is interested in updating these for the default?

Reply:
Uh, yeah sure. does it cause a major issue with other layouts(or add a lot of size to the firmware)? It took me a little bit to figure out what was going on with my caps lock key. I'll make a revision to my keymap telling potential users to re-enable locking caps lock if it does cause an issue/is too big.